### PR TITLE
Fix for omp bug found in the github actions

### PR DESCRIPTION
**Pull request by mvertens**

### CODE REVIEW

**Review by JorgSchwinger — APPROVED** [review]
Excellent, this makes sense, thank you!

**Comment by jmaerz** [comment]
@JorgSchwinger and @mvertens , can we first do/finalize the PR #305 to get a better overview on the applied changes?

**Comment by mvertens** [comment]
@jmaerz - yes that's fine. I need to finalize the minor changes from @jmaerz.

**Comment by JorgSchwinger** [comment]
You can select to view only the last two commits, which worked fine for me to see only the w***_const updates

File: hamocc/mo_param_bgc.F90 [hunk 1]
```diff
@@ -221,15 +241,13 @@ module mo_param_bgc
...
-
   !********************************************************************
-  !     Sinking parameters
+  ! Sinking parameters
   !********************************************************************
-  real :: wpoc                  =  5.             ! m/d   Sinking speed of detritus iris : 5.
-  real :: wcal                  = 30.             ! m/d   Sinking speed of CaCO3 shell material
-  real :: wopal                 = 30.             ! m/d   Sinking speed of opal iris : 60
-  real :: wdust                                   ! m/d   Sinking speed of dust
+  real :: wpoc_const            =  5.             ! m/d   Sinking speed of detritus iris : 5.
```
Comment (jmaerz) [inline hamocc/mo_param_bgc.F90]: These real parameters could now be made `protected`.

Comment (mvertens) [reply hamocc/mo_param_bgc.F90]: Good idea! Just added the protected attribute.

**Comment by jmaerz** [comment]
Dear @mvertens , I felt so free to merge in #305. Now it seems as if it would be good to merge `master` into your `omp_bug_fix` branch to update it - so that only real differences appear.

**Comment by mvertens** [comment]
I think its fine to go ahead and merge this!

**Comment by jmaerz** [comment]
@mvertens then go ahead and take the risk. I don't understand, why there are so many file differences showing up and was also a bit hesitant wrt to your latest changes in #305 (of which I am uncertain, whether they are in or not). But you're probably right to merge it.

**Comment by mvertens** [comment]
So I merged this branch to master and then pushed back. Now only 3 files are different. I feel much better. Lets make sure that the CI tests pass and then I'll merge.

**Review by jmaerz — APPROVED** [review]

**Comment by jmaerz** [comment]
Tomas is busy these days - I thus removed him as reviewer.

**Comment by jmaerz** [comment]
@mvertens many thanks!

**Comment by jmaerz** [comment]
@JorgSchwinger , with this latest merge, `master` is ready for some further styling from your side, Jörg :-)

**Pull request merged**